markhor_description/launch/display_markhor.launch

```diff
@@ -1,13 +1,5 @@
 <launch>
     <arg name="gui" default="true" />
     <!-- load the urdf -->
-    <param name="robot_description" command="xacro '$(find markhor_description)/urdf/markhor.urdf.xacro'" />
-
-    <!-- Allow to spaw the UI to control the flipper and motor values -->
-    <param name="use_gui" value="$(arg gui)"/>
-
-    <!-- Launch rviz to see the model -->
-    <group if="$(arg gui)">
-        <node name="rviz" pkg="rviz" type="rviz" args="-d $(find markhor_description)/rviz/config.rviz"/>
-    </group>
+    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find markhor_description)/rviz/config.rviz"/>
```

Comment:
File name logic is odd. The difference between display_markhor and display_markhor_standalone is lost to me.